cmd/pr-reminder/main.go

```diff
@@ -346,6 +351,16 @@ func findPrsForUsers(users map[string]user, ghClient prClient) map[string]user {
 	return users
 }
 
+func filterLabels(labels []github.Label) []string {
```

Comment:
this function can take an `interestedLabels sets.String` parameter so that it can easily be checked with `interestedLabels.Has(label.Name)`